app/src/main/java/ch/epfl/sweng/swenggolf/TestMode.java

```diff
@@ -0,0 +1,59 @@
+package ch.epfl.sweng.swenggolf;
+
+/**
+ * Class Used to know if we are running a test or not.
+ */
+public class TestMode {
+
+    private TestMode(){}
+
+    /*
+    *   TEST_MODE must be true for tests and false otherwise.
+    */
+    private static boolean TEST_MODE = false;
```

Comment:
Since TEST_MODE is static but not `final`, i.e. not a constant, I would have still written it as testMode.

Reply:
Thanks, I alwaws make his mistake!